libs/orchestrator-ui-components/src/lib/components/Subscription/SubscriptionGeneral.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+    FixedInputBlock,
+    SubscriptionBlock,
+    SubscriptionContext,
+} from '@orchestrator-ui/orchestrator-ui-components';
+import {
+    EuiFlexGrid,
+    EuiFlexItem,
+    EuiLoadingContent,
+    EuiSpacer,
+} from '@elastic/eui';
+
+export const SubscriptionGeneral = () => {
+    const { subscriptionData, loadingStatus } =
+        React.useContext(SubscriptionContext);
+
+    console.log('Loading status: ', loadingStatus);
```

Comment:
`console.log()` statement needs to be removed since it does not belong to production code.